src/nplinker/pairedomics/downloader.py

```diff
@@ -670,42 +685,20 @@ def _parse_genome_labels(self, met_records, gen_records):
                 strain.add_alias(alias)
             self.strains.add(strain)
 
-    def _download_metabolomics_zipfile(self, gnps_task_id):
-        url = GNPS_DATA_DOWNLOAD_URL.format(gnps_task_id)
-
-        self.metabolomics_zip = os.path.join(self.project_download_cache,
-                                             'metabolomics_data.zip')
+    def _download_metabolomics_zipfile_v2(self, gnps_task_id):
```

Comment:
remove this `_v2` function and directly put the new implementation in the function `download_metabolomics_zipfile`